### PR TITLE
Plat UI 1011 rehome using latest libraries

### PR DESCRIPTION
**Pull request by matthewmascord**
This PR migrates the three static content pages in help-frontend, privacy, T&Cs and cookies, to a completely new microservice template based on https://github.com/hmrc/init-service and https://github.com/hmrc/platform-example-ui-journey-tests. Because the changes are so extensive, I don't recommend reviewing as a diff and instead:

- Check out branch and review code in IDE
- Run the unit and acceptance tests, verify all tests have been migrated from the original and are passing.
- Review the three views compared with the original
- Review the messages files
- Spin up locally and compare with pages in production
- Read through original code to verify all functionality has been migrated

**Changes from the original**

When running through the WAVE browser plugin, some warnings relating to missing table captions were reported. These have been added via the govukTable caption argument. This has resulted in an increase to the font weight of the caption due to the default styling from GDS having been applied.

No Axe violations are reported using the Chrome Axe plugin beyond known colour contrast issues.

The external links have been uplifted to follow the 'opens in new tab' guidance from GDS.

Straight quotes have been converted to curly quotes.

**Outstanding Issues**

The privacy page is not actually accessible in production, it redirects to https://www.gov.uk/government/publications/data-protection-act-dpa-information-hm-revenue-and-customs-hold-about-you - recommend this page is removed as it is effectively dead code and has the potential to misdirect. Once that is done, we should remove any internal links to /help/privacy and replace with the GOV.UK link.

URLs probably belong in AppConfig rather than messages files.

There is no 'Accessibility statement' link, linking to www.gov.uk's accessibility statement - this requires a fix to the hmrcStandardHeader component in hmrc/play-frontend-hmrc.  A separate ticket has been created for this.

Using CSPFilter is best practice for Play 2.7+. However, there is a console error because the hmrcScripts and hmrcHead template need updating to support using a CSP Nonce. A separate ticket has been created for this.

The content is out of date and due for a review - this will be done as a separate change.

The 'Get help with this page' link has been replaced with the new 'Is this page not working properly?' link to the standalone form hosted by contact-frontend.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,33 +1,66 @@
-import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, integrationTestSettings}
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
-val appName = "help-frontend"
+val appName = "help-frontend-new"
```
Comment (JoPintoPaul) [inline build.sbt:4]: This should just be `help-frontend`

**Comment by JoPintoPaul** [comment]
Not a big issue but I notice that almost all the messages keys start with `help.`, and it feels redundant to have the application name in the message keys. However, it's not a big issue, and we can live with it.

File: conf/application.conf [hunk 2]
```diff
@@ -12,25 +12,65 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include "common.conf"
+include "frontend.conf"
 
-application.global = config.ApplicationGlobal
-application.langs = "en,cy"
+appName = "help-frontend-new"
```
Comment (JoPintoPaul) [inline conf/application.conf:17]: Again, should this just be `help-frontend`?

File: test/unit/controllers/HelpControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    "return 200" in {
+      val result = controller.cookies(fakeRequest)
+      status(result) mustBe Status.OK
+    }
+
+    "return HTML" in {
+      val result = controller.cookies(fakeRequest)
+      contentType(result) mustBe Some("text/html")
+      charset(result) mustBe Some("utf-8")
+    }
```
Comment (JoPintoPaul) [inline test/unit/controllers/HelpControllerSpec.scala:52]: Worth checking that it is the expected page via the HTML...?

**Comment by JoPintoPaul** [comment]
Running the zap tests locally, I had one failure:
`uk.gov.hmrc.zap.ZapException: Request to endpoint: http://localhost:11000/json/core/view/urls failed with exception: Remotely closed. Check if the service can handle requests.`

File: test/unit/controllers/HelpControllerSpec.scala (hunk 3 above)
Comment (matthewmascord) [reply test/unit/controllers/HelpControllerSpec.scala:52]: Good call 👍

File: app/uk/gov/hmrc/helpfrontend/views/Layout.scala.html [hunk 4]
```diff
@@ -25,6 +26,7 @@
         hmrcStandardHeader: HmrcStandardHeader,
         hmrcStandardFooter: HmrcStandardFooter,
         languageSelect: LanguageSelect,
+        standardBetaBanner: StandardBetaBanner,
```
Comment (JoPintoPaul) [inline app/uk/gov/hmrc/helpfrontend/views/Layout.scala.html:29]: Do you want to use this phase banner? Or is it being injected by mistake?

Comment (matthewmascord) [reply app/uk/gov/hmrc/helpfrontend/views/Layout.scala.html:29]: oops, good catch

**Review by JoPintoPaul — APPROVED** [review]

**Comment by matthewmascord** [comment]
retest this please

**Comment by matthewmascord** [comment]
retest this please

**Comment by matthewmascord** [comment]
retest this please

**Pull request merged**